File: python/examples/example1.py
import os, sys
sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) #Need to find help_handler.py file in the parent directory
from help_handler import *




def main():
    HelpHandler.handle("usage: help dialogue example") #Set help dialogue and process arguments


if __name__ == "__main__":
    main()

File: cpp/examples/example1.cpp
#include "../helpHandler.hpp"


#include <exception>
#include <iostream>




int main(int argc, char** argv) {
    try {
        int rval = helpHandler::handle(argc, argv, "usage: help dialogue example"); //Set help dialogue and process arguments
    } catch(const std::exception& e) { std::cout << e.what(); }
    
    return EXIT_SUCCESS;
}

